src/main.jsx

```
@@ -1,16 +1,176 @@
+              border: 'none',
+              borderRadius: '4px',
+              cursor: 'pointer'
+            }}
+          >
+            Reload Page
+          </button>
+        </div>
+      );
+    }
+
+    return this.props.children;
```

ESLint

[error] 'children' is missing in props validation

(react/prop-types)